Wizardry/Core/BattleSys/Source/BattleCalcReal.c

```diff
@@ -166,6 +166,14 @@ void ComputeBattleUnitEffectiveStats(struct BattleUnit * attacker, struct Battle
     ComputeBattleUnitSilencerRate(attacker, defender);
     ComputeBattleUnitSpecialWeaponStats(attacker, defender);
 
+#if (defined(SID_NoGuard) && (SID_NoGuard < MAX_SKILL_NUM))
+        if (SkillTester(&attacker->unit, SID_NoGuard))
```

Comment:
better to put into ComputSkill func

Reply:
Respectively, this is starting to get a bit silly now. This is the third time you've asked me to change where to put it. Please make up your mind. 

I've put it in the `ComputSkill` function now.